storm/src/main/storm/mesos/NimbusScheduler.java

```diff
@@ -102,7 +102,6 @@ public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
       case TASK_LOST:
       case TASK_ERROR:
         LOG.info("Received status update: {}", taskStatusToString(status));
```

Comment:
TASK_ERROR has now the same action as TASK_RUNNING, so is TASK_STARTING. Isn't it misleading to have state of the task set to TASK_RUNNING when it has an error (TASK_ERROR) or is starting (TASK_STARTING)? This state is being set by setState() and I do not see state ever being set to TASK_ERROR or TASK_STARTING. Wondering if TASK_ERROR and TASK_STARTING are unused and untouched states.

```
12:58 PM sagrawal🐾  mesos-storm [master]$: git grep setState
storm/src/main/storm/mesos/MesosSupervisor.java:        .setState(TaskState.TASK_FINISHED)
storm/src/main/storm/mesos/MesosSupervisor.java:            .setState(TaskState.TASK_FAILED)
storm/src/main/storm/mesos/MesosSupervisor.java:          .setState(TaskState.TASK_RUNNING)
```